.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- 
+
+Sorry to hear you're having trouble! If you have a question, please ask in 
+[SO or gitter](https://github.com/callemall/material-ui/blob/master/CONTRIBUTING.md#asking-questions)
```

Comment:
Yeah, I was wondering too whether it's important to have link context. As it relates to url shortener, I don't think I have a preference :smile:, if I recognize it, I'm good!

I suppose the right long-term way to do this is to setup routes on our domain and we can set up a page or redirect them wherever we want. Something like:

https://material-ui.com/contributing -> github contributing md
https://material-ui.com/stackoverflow -> stack overflow page
https://material-ui.com/gitter -> open gitter in tab to sign up or our group

They're not super short but short enough for this use case.